validation/jinja2/validation/_segment_card.html

```diff
@@ -18,22 +18,27 @@
 {% import 'validation/_macros.html' as macros %}
 {% import 'validation/_issue_modal.html' as issue_modal %}
 {% import 'validation/_speaker_code_modal.html' as speaker_code_modal %}
+{% import 'validation/_wrong_buttons.html' as wrong_buttons %}
 
 {% macro segment_card(phrase, recordings, speakers, is_linguist, is_expert, auth, forms, csrf_token, request) %}
 
-<article class="card shadow-sm" data-cy="segment-card">
+    <article id='{{ phrase.id }}' class="card shadow-sm" data-cy="segment-card">
```

Comment:
```suggestion
    <article id='phrase-{{ phrase.id }}' class="card shadow-sm" data-cy="segment-card">
```

You’d have to change the references as well, but an HTML id that looks like it’s purely a number might cause confusion down the road.